bin/lint

```diff
@@ -2,3 +2,4 @@
 
 black --check --diff .
 ruff check .
+black --check
```

Comment:
huh, it looks like line 3 should already be running black --check. I'm guessing what's occurred is that the CI run was reformatting before lint. this line can now be removed.